dart/lib/src/browser_client.dart

```diff
@@ -41,19 +42,19 @@ class SentryBrowserClient extends SentryClient {
     );
   }
 
-  SentryBrowserClient._(SentryOptions options, {String origin, String platform})
+  SentryBrowserClient._(SentryOptions options,
+      {String origin, @required String platform})
       : super.base(
           options,
-          origin: origin,
-          sdk: Sdk(name: browserSdkName, version: sdkVersion),
-          platform: platform,
+          transport: Transport(
+            dsn: options.dsn,
+            httpClient: options.httpClient,
+            clock: options.clock,
+            compressPayload: false,
+            sdk: Sdk(name: browserSdkName, version: sdkVersion),
```

Comment:
`options.sdkVersion` already exists, but its probably null, we'd need to find a way to instantiate it based on the platform (dart, flutter or web), but ideally, we just read this field from the options here

Reply:
on a second thought, shouldn't this be just either: `sentry.dart` or `sentry.flutter`? Given that this is what the user installed at the end?

Reply:
if we need only the identifier of the client, yes, only a String would be fine.

Reply:
✅ fixed